src/pybind/mgr/dashboard/run-frontend-i18n.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd frontend
+
+if [ `uname` != "FreeBSD" ]; then
+  . $CEPH_ROOT/build/src/pybind/mgr/dashboard/node-env/bin/activate ||
+  echo 'Info: missing node-env'
+fi
+
+npm run i18n
+
+count=`git status | grep "messages.xlf" -c`
+
+if [ $count == 1 ]; then
+  echo  "Please run 'npm run i18n' and commit the modified 'messages.xlf' file."
```

Comment:
At this point `npm run i18n` has already been run and `messages.xlf` is up to date, isn't it?
```suggestion
  echo  "The translation template file 'messages.xlf' has been modified. Please commit the changes."
```

Reply:
This script is more focused in being used in jenkins, so this message is going to be read after a failing `ceph dashboard tests`. Then the developer will need to run the script locally.